requirements.txt

```
@@ -11,3 +11,5 @@ torch==1.7.1+cu101
 # torch-xla is a bit of a mess. configure as needed on your machine.
 https://storage.googleapis.com/tpu-pytorch/wheels/torch_xla-1.7-cp36-cp36m-linux_x86_64.whl
 torchvision==0.8.2+cu101
+tensorboard
+cv2==4.5.1
```

We're not using `requirements.txt` anymore, but the correct package name is `opencv-python`
```suggestion
opencv-python==4.5.1
```